subsystems/win/basesrv/dosdev.c

```diff
@@ -13,9 +13,21 @@
 #define NDEBUG
 #include <debug.h>
 
+typedef struct _BSM_REQUEST
+{
+    struct _BSM_REQUEST * Next;
+    LUID BroadcastLuid;
+    LONG DriveLetter;
+    LONG RemoveDefinition;
+} BSM_REQUEST, *PBSM_REQUEST;
+
 /* GLOBALS ********************************************************************/
 
 static RTL_CRITICAL_SECTION BaseDefineDosDeviceCritSec;
+RTL_CRITICAL_SECTION BaseSrvDDDBSMCritSec;
+PBSM_REQUEST BSM_Request_Queue = NULL, BSM_Request_Queue_End = NULL;
+ULONG BaseSrvpBSMThreadCount = 0;
+LONG (WINAPI *PBROADCASTSYSTEMMESSAGEEXW)(DWORD, LPDWORD, UINT, WPARAM, LPARAM, PBSMINFO) = NULL;
```

Comment:
This name looks like it should be a typedef. `pBroadcastSystemMessageExW` instead?